satellite/audit/verifier.go

```diff
@@ -359,6 +368,13 @@ func (verifier *Verifier) Reverify(ctx context.Context, path storj.Path) (report
 		}
 		return Report{}, err
 	}
+	if pointer.ExpirationDate != (time.Time{}) && pointer.ExpirationDate.Before(time.Now().UTC()) {
+		errDelete := verifier.metainfo.Delete(ctx, path)
```

Comment:
This might delete valid data.

Reply:
if the data is expired, how could it be valid?